libsrc/telestrat/kbhit.s

```diff
@@ -10,12 +10,10 @@
 
 _kbhit:
         BRK_TELEMON XRD0
-        bcs     @no_char_action
         ldx     #$00 
+        bcs     @no_char_action
         lda     #$01
         rts
 @no_char_action:
```

Comment:
Does A have a defined value at this point from BRK_TELEMON XRD0? If yes, it might be worth mentioning in a comment.

Reply:
It uses carry to set return value, but i have to look to source code to know what A returns. I try to have a look tonight

Reply:
Sorry, my comment above THAT useful. What I actually wanted to express and what I probably should have written is: "I consider your recent proposal broken. The only way it could potentially work is if BRK_TELEMON XRD0 would be well known to return a certain value in A that you only have to override if necessary. But I rather presume that this isn't the case." You see, I'm not actually interested in the fact what BRK_TELEMON XRD0 returns in A. There's no need that you try to find out. Rather just go ahead and write code here which works independently from BRK_TELEMON XRD0' setting of A.

Reply:
Sorry, Oliver, I cannot follow you.